frontend/app/services/AuthenticationService.scala

```diff
@@ -59,14 +58,14 @@ object AuthenticationService {
     // Target client not relevant; only applicable to crypto access tokens,
```

Comment:
Could you also remove this comment? Since you’ve gotten rid of the `targetClient` argument on the line below, I don’t think this is necessary anymore.